training-react/src/app/components/LoadingScreen/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import LoadingItem from '../LoadingItem';
+
+function withLoadingScreen(WrappedComponent) {
+  return function LoadingScreen({ isLoading, ...otherProps }) {
+    if (isLoading) return <LoadingItem />;
+    return <WrappedComponent {...otherProps} />;
```

Comment:
Just as a comment (don't need to change it)
you can achieve the same in one line
```js
return isLoading ? <LoadingItem /> : <WrappedComponent {...otherProps} />;
```

Reply:
Cool :+1: